config/routes.rb

```diff
@@ -1,3 +1,11 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/show', to: 'insights#show'
+  get '/contributors', to: 'insights#_contributors'
```

Comment:
InsightsController has only one method #show. Does these routes work? 
Also, where did you get this naming starting from '_' ?

Reply:
Working well! It give you a chance to name your route in a way you want. Got it here: https://www.railstutorial.org/book/filling_in_the_layout